packages/react-components/src/AccessibilityScenarios/Button.stories.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+
+// https://github.com/microsoft/fluentui/pull/18695#issuecomment-868432982
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+import { Button } from '@fluentui/react-button';
+import { Scenario, APP_TITLE, APP_TITLE_SEPARATOR } from './utils';
+
+export const CalendarButtonsAccessibilityScenario: React.FunctionComponent = () => {
+  const onSaveButtonClick = React.useCallback(() => {
+    alert('The calendar event would have been saved.');
```

Comment:
Could we use `console.log()` here instead? `alert()` is a bit disruptive.

Reply:
Or rather just build some kind of status update in the UI...

Reply:
I have replaced the alert() with a &lt;p&gt; element containing the event status text.